Explorer/Assets/DCL/AvatarRendering/Emotes/Emote.cs

```diff
@@ -17,6 +17,12 @@ public class Emote : IEmote
 
         public bool IsLoading { get; set; } = true;
 
+        public bool IsOnChain()
+        {
+            var id = ((IAvatarAttachment) this).GetUrn().ToString();
+            return id.StartsWith("urn:") && !id.StartsWith("urn:decentraland:off-chain:");
```

Comment:
Maybe I'm just tired, but wouldn't the second condition already cover the first one? We're comparing twice something we can do once, just like you did for `Wearable.cs`. What do you think?

Also, what do you think about turning the literals to constants?

Reply:
I belive its not the same, you can check the inputs in the unit tests